src/store/plugins/confirmations.js

```diff
@@ -0,0 +1,33 @@
+import Vue from 'vue';
+import { i18n } from '../../popup/utils/i18nHelper';
+
+const confirmationsText = {
+  login: i18n.t('pages.confirmations.login'),
+  messageSign: i18n.t('pages.confirmations.message-sign'),
+  transactionSign: i18n.t('pages.confirmations.transaction-sign'),
+};
+
+export default store =>
+  store.registerModule('confirmations', {
+    namespaced: true,
+    state: {
+      undeniable: {
```

Comment:
is this nesting level necessary? I think it can be just `state.permissions.address`, `state.permissions.transactionSign`, `state.permissions.messageSign`